app/services/dwp_monitor.rb

```diff
@@ -25,8 +25,13 @@ def percent
     # When DWP is offline it returns 400 Bad Request
     # maybe extend to search for x00 as first 3 chars to check for 500 errors too
     total = @checks.count.to_f
+    error_total / total * 100.0
```

Comment:
I always like to include brackets to indicate precedence explicitly - but that's just me